### PR TITLE
Test80 (Python3)

### PR DESCRIPTION
**Pull request by manics**
Related PRs:
- [x] https://github.com/ome/ansible-role-omero-web-apps/pull/7
- [x] https://github.com/ome/omego/pull/126
- [x] https://github.com/ome/ansible-role-omero-server/pull/40
- [x] https://github.com/ome/omero-py/pull/110
- [x] https://github.com/ome/ansible-role-ice/pull/11
- [x] https://github.com/ome/ansible-role-omero-web/pull/26

Currently broken:
- [x] iviewer (can't be installed)
- [x] gallery root redirect is disabled
- [x] `idr-reset-users-groups.yml` Can't reset password https://github.com/ome/omero-py/issues/112
- [x] Assumes Python 2 server deployment `lib/python` https://github.com/ome/ansible-role-omero-prometheus-exporter
- [x] Assumes Python 2 web deployment `venv` https://github.com/ome/ansible-role-omero-web-django-prometheus
- [x] ome.prometheus broken for mysterious reasons

Works but still on Python 2:
- [ ] https://github.com/ome/ansible-role-omero-logmonitor (or drop since we've got also got fluentd)

Note yet deployed:
- [x] Grafana untested
- [x] Fluentd untested

Fails idempotent check:
- [x] OMERO.server upgrade (version not compared)
- [x] [`Set Nginx proxy timeout`](https://github.com/IDR/deployment/blob/IDR-0.7.1/ansible/idr-omero-web.yml#L32-L41): https://github.com/ome/omero-web/issues/65

### CODE REVIEW

**Comment by manics** [comment]
`https://github.com/ome/ansible-role-omero-logmonitor (or drop since we've got also got fluentd)` is the last remaining issue. This runs in an independent python 2 environment (it doesn't depend on omero-py) so it can be removed in a future release.

File: ansible/idr-omero-web.yml [hunk 1]
```diff
@@ -6,43 +6,19 @@
   roles:
   - role: ome.redis
   - role: ome.omero_web
-  - role: ome.omero_web_apps
 
   # Vars are in group_vars/omero-hosts.yml
 
   tasks:
+  # TODO: port https://github.com/ome/openmicroscopy/pull/6094
```
Comment (joshmoore) [inline ansible/idr-omero-web.yml]: This is excluded and on the wrong repo now, no?

File: ansible/idr-reset-users-groups.yml [hunk 2]
```diff
@@ -19,6 +19,15 @@
     set_fact:
       omero_db_host_ansible: "{{ hostvars[groups[idr_environment | default('idr') + '-database-hosts'][0]]['ansible_' + (idr_net_iface | default('eth0'))]['ipv4']['address']}}"
 
+  # TODO: Remove when https://github.com/ome/omero-py/pull/175 is released
```
Comment (joshmoore) [inline ansible/idr-reset-users-groups.yml]: Could be done today, I think.

File: ansible/requirements.yml [hunk 3]
```diff
@@ -73,40 +73,39 @@
...
 
+# https://github.com/ome/ansible-role-omero-server/pull/40
 - src: ome.omero_server
-  version: 2.1.1
+  version: 3.1.0
 
 - src: ome.omero_user
   version: 0.1.2
 
+# https://github.com/ome/ansible-role-omero-web/pull/26
```
Comment (joshmoore) [inline ansible/requirements.yml]: These comments probably aren't needed anymore, eh?

File: ansible/idr-omero-web.yml (hunk 1 above)
Comment (manics) [inline ansible/idr-omero-web.yml]:
```suggestion
  # TODO: https://github.com/ome/omero-web/pull/123
```

File: ansible/requirements.yml (hunk 3 above)
Comment (manics) [inline ansible/requirements.yml]:
```suggestion
```

File: ansible/requirements.yml [hunk 4]
```diff
@@ -73,40 +73,39 @@
...
-- src: ome.omego
-  version: 0.1.3
-
 - src: ome.omero_logmonitor
   version: 2.1.1
 
 - src: ome.omero_python_deps
   version: 2.0.1
 
+# https://github.com/ome/ansible-role-omero-server/pull/40
```
Comment (manics) [inline ansible/requirements.yml]:
```suggestion
```

**Comment by manics** [comment]
omero-logmonitor still works the same as always, it's on Python 2 (and as mentioned doesn't require omero-py), but I think the fluentd alerts mean it's no longer needed. OTOH it's already deployed so best to remove it in a later release.

**Comment by manics** [comment]
Updated with omero-py==5.6.1. Merging for prod80 deployment.

**Pull request merged**